apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -76,20 +72,15 @@ export default function SchoolDataInputs({
   return (
     <div className={style.schoolAssociationWrapper}>
       {includeHeaders && (
-        <div>
-          <Heading2 className={style.topPadding}>
-            {i18n.censusHeading()}
-          </Heading2>
+        <div className={style.headerContainer}>
+          <Heading2>{i18n.censusHeading()}</Heading2>
           <BodyTwoText>{i18n.schoolInfoInterstitialTitle()}</BodyTwoText>
         </div>
       )}
       <div className={style.inputContainer}>
-        <BodyThreeText className={style.padding}>
-          <strong>{i18n.whatCountry()}</strong>
-        </BodyThreeText>
         <SimpleDropdown
-          id="uitest-country-dropdown"
```

Comment:
None of these `uitest` ids were being used anywhere so I removed them, but let me know if we will want these!